sdk/main/src/Executable.cc

```diff
@@ -0,0 +1,83 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include "Executable.h"
+
+#include "AccountBalance.h"
+#include "AccountBalanceQuery.h"
+#include "AccountCreateTransaction.h"
+#include "Client.h"
+#include "Node.h"
+#include "TransactionResponse.h"
+
+#include <proto/query.pb.h>
+#include <proto/response.pb.h>
+#include <proto/transaction.pb.h>
+#include <proto/transaction_response.pb.h>
+
+#include <grpcpp/impl/codegen/status.h>
+
+#include <utility>
+
+namespace Hedera
+{
+//-----
+template<typename SdkRequestType, typename ProtoRequestType, typename ProtoResponseType, typename SdkResponseType>
+SdkResponseType Executable<SdkRequestType, ProtoRequestType, ProtoResponseType, SdkResponseType>::execute(
+  const Client& client)
+{
+  return execute(client, std::chrono::minutes(2L));
```

Comment:
Is this literal documented anywhere?

Reply:
I changed the functionality of this in another PR (using `Client.getRequestTimeout()` instead of the literal) and left a comment in there saying the default is 2 minutes.